region/norfair/west/Crocomire Speedway.json

```diff
@@ -548,6 +548,90 @@
       ],
       "devNote": "FIXME: R-Mode can be used instead of a Crystal Flash but that depends on having more Reserve Energy than Regular Energy."
     },
+    {
+      "link": [2, 2],
+      "name": "Multiviola Pirate Farm",
+      "requires": [
+        "h_heatProof",
+        {"resetRoom": {"nodes": [2]}},
+        {"cycleFrames": 1200},
+        {"or": [
+          "Plasma",
+          {"and": [
+            "Wave",
+            {"cycleFrames": 120}
+          ]},
+          {"and": [
+            "Spazer",
+            {"cycleFrames": 300}
+          ]},
+          {"and": [
+            "Grapple",
+            {"cycleFrames": 45}
+          ]},
+          {"and": [
+            "canPseudoScrew",
+            {"cycleFrames": 220}
+          ]},
+          {"cycleFrames": 120}
+        ]}
+      ],
+      "farmCycleDrops": [
+        {"enemy": "Multiviola", "count": 4},
+        {"enemy": "Red Space Pirate (standing)", "count": 3}
+      ],
+      "resetsObstacles": ["A"],
+      "devNote": "Some frames added to account for spike RNG."
```

Comment:
Copy-paste errors from the Cacatac farm? Multiviolas and Red Space Pirates are invulnerable to Grapple. Red Space Pirates are invulnerable to Power Beam. And there's no spike RNG?

Reply:
Goota be.
This is probably where I started testing speedbooster, then forgot to finish up beam farms.